lib/orca_api/client.rb

```diff
@@ -353,7 +357,7 @@ def new_http
       http
     end
 
-    def make_request(http_method, path, params, body, format)
+    def make_request(http_method, path, params, body, format, type)
```

Comment:
typeが使われていません。
何を入れるのですか？

Reply:
Because this is an XML request, I don't want to parse the body using JSON. I tried sending it as a json string, but the API said it couldn't read the arguments. I then sent it using xml instead, and since I don't want my body to be parsed into json when I send it in xml, I added a type here to make sure it is still xml.